app/src/js/components/Collections/list.js

```diff
@@ -103,7 +98,10 @@ class CollectionList extends React.Component {
         </section>
         <section className='page__section'>
           <div className='heading__wrapper--border'>
-            <h2 className='heading--medium heading--shared-content with-description'>{strings.all_collections} <span className='num--title'>{count ? ` ${tally(count)}` : 0}</span></h2>
+            <h2 className='heading--medium heading--shared-content with-description'>
+              {hasTimeFilter ? 'Active Collections' : strings.all_collections}
```

Comment:
nit: consider adding `'Active Collections'` to the `strings` object for consistency.

Reply:
Sure thing